test/server/parser/parser.ts

```diff
@@ -41,12 +69,13 @@ function toBeValidSyntax(file: string)
 declare module 'expect'
 {
 	interface Matchers<R>
-	{
+{
```

Comment:
Is this de-indent intentional? it looks wrong

Reply:
That's definitely wrong. Looks like eslint auto formatter got confused